LoginPage.dart

```diff
@@ -0,0 +1,218 @@
+import 'package:flutter/material.dart';
+import 'RegistrationPage.dart';
+import 'PostList.dart';
+import 'package:http/http.dart' as http;
+import 'dart:convert';
+
+String? globalToken;
+
+class LoginPage extends StatefulWidget {
+  @override
+  State<LoginPage> createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  TextEditingController emailController = TextEditingController();
+  TextEditingController passwordController = TextEditingController();
+  bool isPasswordVisible = false;
+  bool _isNotValidate = false;
+  bool _isHovering = false;
+
+  bool _validateEmail(String value) {
```

Comment:
Kad atskirti funkcijas nuo kintamųjų bei kad išlaikyti vieningumą tarp funkcijų ir metodų pavadinimų, geriau išlaikyti pavadinimų formatą (camelCase) validateEmail, be jokių papildomų simbolių